mathesar/api/exceptions/exceptions.py

```diff
@@ -0,0 +1,7 @@
+from collections import namedtuple
+
+
+ExceptionBody = namedtuple('ExceptionBody',
+                           ['code', 'message', 'field', 'details'],
+                           defaults=[None, None]
+                           )
```

Comment:
```suggestion

ExceptionBody = namedtuple(
    'ExceptionBody',
    ['code', 'message', 'field', 'details'],
    defaults=[None, None]
)
```

Reply:
I find the first one to be easier to understand, it looks like these belong to the function